Sources/Switch.swift

```diff
@@ -0,0 +1,96 @@
+private let frameSize = CGSize(width: 51, height: 21)
+private let thumbSize = CGFloat(31.0)
+private let animationTime = 0.25
+
+private let defaultOnTintColor = CGColor(red: 169 / 255, green: 218 / 255 , blue: 214 / 255, alpha: 1)
+private let defaultThumbColor = CGColor(red: 14 / 255, green: 136 / 255 , blue: 122 / 255, alpha: 1)
+
+private let offThumbColor = CGColor(red: 176 / 255, green: 176 / 255 , blue: 176 / 255, alpha: 1)
+private let offTrackColor = CGColor(red: 225 / 255, green: 225 / 255 , blue: 225 / 255, alpha: 1)
```

Comment:
nit: personally I would move these private styling vars to the bottom of the file